api4/user_test.go

```diff
@@ -5515,6 +5515,51 @@ func TestGetThreadsForUser(t *testing.T) {
 		require.NotNil(t, uss3.Threads)
 		require.Len(t, uss3.Threads, 0)
 	})
+
+	t.Run("editing or reacting to reply post does not make thread unread", func(t *testing.T) {
+		Client := th.Client
+
+		rootPost, _ := postAndCheck(t, Client, &model.Post{ChannelId: th.BasicChannel.Id, Message: "root post"})
```

Comment:
Shall we check the response errors?

Reply:
The `postAndCheck` function checks the response for errors. I'm not sure why `postAndCheck` also returns the response, it isn't used anywhere.